packages/apps/plugins/plugin-thread/src/components/MessageContainer.tsx

```diff
@@ -106,12 +107,9 @@ const TextboxBlock = ({
   useOnEditAnalytics(message, editing);
 
   return (
-    <div
-      role='none'
-      className={mx('col-span-3 grid grid-cols-subgrid', hoverableControls, hoverableFocusedWithinControls)}
-    >
-      <div ref={parentRef} className={textboxWidth} {...focusAttributes} />
-      <div role='none' className='flex flex-row items-center'>
+    <div role='none' className={mx('col-span-3', hoverableControls, hoverableFocusedWithinControls)}>
```

Comment:
This needs to have `grid-cols-subgrid` so its children maintain alignment to the grid. Why is it dropped here?